src/main/python/pybuilder/pip_utils.py

```diff
@@ -21,14 +21,11 @@
 import sys
 
 from pybuilder.core import Dependency, RequirementsFile
-from pybuilder.pip_common import (Version,
-                                  SpecifierSet,
-                                  search_packages_info,
-                                  pip_working_set_init,
-                                  _pip_disallows_insecure_packages_by_default,
-                                  _pip_supports_constraints,
-                                  pip_version)
 from pybuilder.utils import execute_command, as_list
+# Plugin install_dependencies_plugin can reload pip_common and pip_utils. Do not use from ... import ...
+from pybuilder import pip_common
+Version = pip_common.Version
```

Comment:
We can guarantee it as reload is only relevant because of a PIP upgrade. At any rate, unless you're reloading `pip_utils` the below does not solve the problem of reload `pip_common` only, doesn't it?

> from pybuilder import pip_common
> Version = pip_common.Version